integration/dubbo-alibaba/src/main/java/io/seata/integration/dubbo/alibaba/TransactionPropagationFilter.java

```diff
@@ -63,14 +68,19 @@ public Result invoke(Invoker<?> invoker, Invocation invocation) throws RpcExcept
         } finally {
             if (bind) {
                 String unbindXid = RootContext.unbind();
+                String unbindFilterType = RootContext.unbindFilterType();
                 if (LOGGER.isDebugEnabled()) {
-                    LOGGER.debug("unbind[" + unbindXid + "] from RootContext");
+                    LOGGER.debug("unbind[{}] from RootContext", unbindXid);
+                    LOGGER.debug("unbind filterType[{}] from RootContext", unbindFilterType);
```

Comment:
merge logs

Reply:
@xingfudeshi pr1813 include these codes and logs merged,can reveiw 1813 first.